examples/single_calculations/example_dft.py

```diff
@@ -11,42 +11,45 @@
 from __future__ import print_function
 from __future__ import absolute_import
 
+import os
 import sys
 import ase.build
 import click
 
 from aiida.engine import run
-from aiida.orm import (Code, Dict, StructureData)
+from aiida.orm import (Code, Dict, SinglefileData, StructureData)
 from aiida.common import NotExistent
 from aiida.plugins import CalculationFactory
 
 Cp2kCalculation = CalculationFactory('cp2k')
 
 
-@click.command('cli')
-@click.argument('codelabel')
-def main(codelabel):
+def example_dft(cp2k_code):
     """Run simple DFT calculation"""
-    try:
-        code = Code.get_from_string(codelabel)
-    except NotExistent:
-        print("The code '{}' does not exist".format(codelabel))
-        sys.exit(1)
 
     print("Testing CP2K ENERGY on H2O (DFT)...")
 
+    pwd = os.path.dirname(os.path.realpath(__file__))
+
     # structure
     atoms = ase.build.molecule('H2O')
     atoms.center(vacuum=2.0)
     structure = StructureData(ase=atoms)
 
+    # basis set
```

Comment:
instead of repeating this for every script, why not have the absolute path to these two files stored somewhere in the plugin and just import it?

Reply:
you could add a `BASIS_MOLOPT_ABS` and `GTH_POTENTIALS_ABS` to some `__init__.py` somewhere and import it in all of your tests.

saves you the explicit construction of the path & makes it easier to change if you move files around.

Anyhow, up to you - feel free to go ahead and merge without me :-)